src/pycatcher/catch.py

```diff
@@ -530,13 +539,13 @@ def generate_outliers_stl(df, type, period) -> pd.DataFrame:
         result = stl.fit()
 
         # Back-transform if Box-Cox was applied
-        trend_transformed = inv_boxcox(result.trend, lambda_)
-        seasonal_transformed = inv_boxcox(result.seasonal, lambda_)
+        # trend_transformed = inv_boxcox(result.trend, lambda_)
+        # seasonal_transformed = inv_boxcox(result.seasonal, lambda_)
         residual_transformed = inv_boxcox(result.resid, lambda_)
 
         # Access the components
-        trend = trend_transformed
-        seasonal = seasonal_transformed
+        # trend = trend_transformed
+        # seasonal = seasonal_transformed
```

Comment:
Not using these variable need to rethink if we need it

Reply:
Updated code to remove it